src/test/java/uk/gov/justice/hmpps/casenotes/controllers/CreateCaseNoteIntTest.kt

```diff
@@ -160,6 +160,27 @@ class CreateCaseNoteIntTest : IntegrationTest() {
     assertThat(event.additionalInformation.syncToNomis).isEqualTo(false)
   }
 
+  @Test
+  fun `successfully strips null character from text before saving`() {
+    val type = getAllTypes().first { !it.syncToNomis }
+    val request = createCaseNoteRequest(type = type.typeCode, subType = type.code, text = "This is some text" + Char.MIN_VALUE)
```

Comment:
Maybe add null at the beginning and middle as well as the end to confirm they're all removed as expected